jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -377,7 +377,7 @@ cassandraBranches.each {
         disabled(false)
         using('Cassandra-template-artifacts')
         axes {
-            if (branchName == 'trunk' || branchName == 'cassandra-4.0') {
+            if (branchName == 'trunk' || !(branchName ==~ 'cassandra-[2-3].\d+')) {
```

Comment:
So if it is not anything 2,3.x

Reply:
yes. and so the trunk comparison can be removed. let me update.